index.html

```diff
@@ -1,200 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>The New York Times</title>
-    <link rel="stylesheet" href="css/style.css">
-    <script src="https://kit.fontawesome.com/3a7b6dc568.js"></script>
-</head>
-<body>
-  <!-- Navbar Navbar -->
-  <navbar class="navbar">
-    <i class="fas fa-bars"></i>
-    <i class="fas fa-search"></i>
-    <h3 class="article-section" >Space & Cosmos</h3>
-    <img id="new-york-times-logo" class="logo small" src="assets\The_New_York_Times_logo.png" alt="New York Times logo">
-    <button class="page-button nav-button" >LOG IN</button>
-    <button class="page-button nav-button" >SUBSCRIBE NOW</button>
-  </navbar>
-  <h1 class="title main container">Space Ripples Reveal<br>Big Bang’s Smoking Gun</h1>
-  <div class="container title-img article-media">
```

Comment:
Use `figure` and `figcaption` to wrap image and its description.

Do the same thing in all the images